gradle-enterprise-conventions-maven-extension/extension/src/main/resources/gradle-enterprise.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+
+<gradleEnterprise
+        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+    <server>
+        <!-- todo: change to your own Gradle Enterprise instance -->
+        <url>https://ge.example.com</url>
+        <allowUntrusted>false</allowUntrusted>
+    </server>
+    <buildScan>
+        <capture>
+            <goalInputFiles>true</goalInputFiles>
+        </capture>
+        <!-- todo: implement this more specifically for your CI environment -->
+        <backgroundBuildScanUpload>#{isFalse(env['CI'])}</backgroundBuildScanUpload>
+        <publish>ALWAYS</publish>
+        <obfuscation>
+            <ipAddresses>#{{'0.0.0.0'}}</ipAddresses>
+        </obfuscation>
+    </buildScan>
+    <buildCache>
+        <local>
+            <enabled>true</enabled>
+            <!-- For short-lived CI agents, it makes sense to disable the local build cache. For example: -->
+            <!-- <enabled>#{isFalse(env['CI'])}</enabled> -->
+        </local>
+        <remote>
+            <enabled>true</enabled>
+            <!-- todo: implement this more specifically for your CI environment -->
+            <storeEnabled>#{isTrue(env['CI'])}</storeEnabled>
+        </remote>
+    </buildCache>
+</gradleEnterprise>
```

Comment:
@clayburn can you please get rid of this file and move the logic to the listener class? This way, all logic will be in code. Similarly to how it is done for the Convention GE Gradle plugin.

Reply:
If we move this to the programmatic configuration, then it is not possible for a project to override any of this via the project's `gradle-enterprise.xml`. Project's using the convention extension would either need to use use programmatic configuration or command line arguments to override these values. Given that, do we want to move this logic to programmatic configuration.

Reply:
Ah, got it. I would lean towards moving it to code. The convention plugin should contain the core configuration that is expected for all projects.

Reply:
If the config is in code, can it be overridden via groovy script?

Reply:
In any event, I will revert back to the code methodology of configuring.